core/src/components/textarea/test/label-placement/textarea.e2e.ts

```diff
@@ -179,6 +179,19 @@ configs().forEach(({ title, screenshot, config }) => {
       const textarea = page.locator('ion-textarea');
       expect(await textarea.screenshot()).toMatchSnapshot(screenshot(`textarea-placement-floating-no-value`));
     });
+    test('label should appear on top of the textarea when there is a placeholder and no value', async ({ page }) => {
```

Comment:
During development I discovered we did not have test coverage for this same behavior on `ion-textarea`, so I added it. `ion-input` has test coverage already.